dali/pipeline/operators/op_spec.h

```diff
@@ -407,17 +493,17 @@ inline bool OpSpec::TryGetArgumentImpl(
     if (ws == nullptr)
       return false;
     const auto& value = ws->ArgumentInput(name);
-    if (!IsType<S>(value.type()))
+    if (!IsType<T>(value.type()))
```

Comment:
Tensor arguments are not promoted to int64 - this was handled by the code in macros; the generic code was dead.